setup.py

```diff
@@ -27,15 +27,24 @@ def c_ext():
 
     # compile for MacOS without openmp
     if sys.platform == 'darwin':
-        return [Extension('skbeam.ext.ctrans', ['src/ctrans.c'])]
+        return [Extension('skbeam.ext.ctrans', ['src/ctrans.c'],
+                          include_dirs=['src'])]
     # compile the extension on Linux.
     return [Extension('skbeam.ext.ctrans', ['src/ctrans.c'],
+                      include_dirs=['src'],
                       extra_compile_args=['-fopenmp'],
                       extra_link_args=['-lgomp'])]
 
 
+# add all cython extensions here
 def cython_ext():
-    return cythonize("**/*.pyx")
+    try:
+        ext = cythonize("skbeam/core/accumulators/histogram.pyx")
```

Comment:
Should this be nested into a list?

Reply:
From @jrmlhermitte:
> `cythonize()` returns a list.

```python
In [1]:  from Cython.Build import cythonize

In [2]: cythonize("histogram.pyx")
Compiling histogram.pyx because it changed.
[1/1] Cythonizing histogram.pyx
Out[2]: [<distutils.extension.Extension('skbeam.core.accumulators.histogram') at 0x152a4168ce10>]
```

Reply:
🐑